user_data/hyperopts/sample_hyperopt.py

```diff
@@ -18,14 +18,19 @@
 class SampleHyperOpts(IHyperOpt):
     """
     This is a test hyperopt to inspire you.
+
     More information in https://github.com/freqtrade/freqtrade/blob/develop/docs/hyperopt.md
-     You can:
-    - Rename the class name (Do not forget to update class_name)
-    - Add any methods you want to build your hyperopt
-    - Add any lib you need to build your hyperopt
-     You must keep:
-    - the prototype for the methods: populate_indicators, indicator_space, buy_strategy_generator,
-    roi_space, generate_roi_table, stoploss_space
+
+    You can:
+    - Rename the class name.
+    - Add any methods you want to build your hyperopt.
+    - Add any lib you need to build your hyperopt.
+
+    You must keep:
+    - The prototypes for the methods: populate_indicators, indicator_space, buy_strategy_generator.
+
+    The roi_space, generate_roi_table, stoploss_space methods were moved to the parent class, you may
```

Comment:
I don't think we ~~need to~~ should mention that these were moved to the parent class ... new users will not know this - so mentioning that these are available, and can be overridden should suffice.

Reply:
I also have had doubts about this phrase...

Reply:
More appropriate and detailed comment added, pls verify.